python-exec/string_based_code.py

```diff
@@ -0,0 +1,19 @@
+code = """
```

Comment:
Though re https://3.basecamp.com/4210154/buckets/12741492/messages/5191959771 maybe we are opting against them altogether? Up to you, since we are still talking about this.